app/views/control_panel/stories/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_for [:control_panel, :board, @story] do |f|%>
```

Comment:
Here it looks much better ! :) and you don't have to use url.
Why don't you create partial : `_form` and use it for both new and edit? 
It you don't know what I mean you can generate rails `scaffolding` and check how rails crew use it ;)